pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>org.alfresco.tas</groupId>
     <artifactId>restapi</artifactId>
     <name>alfresco-tas-restapi</name>
-    <version>1.65-SNAPSHOT</version>
+    <version>1.66</version>
```

Comment:
You shouldn't need to change the version manually.

Reply:
Done